client/src/controllers/ActionController.ts

```diff
@@ -0,0 +1,42 @@
+import { Controller } from '@hotwired/stimulus';
+
+//  <button data-controller="w-action"
+//  data-w-action-csrf-token-value = '{{ csrf_token|escapejs }}'
+//  data-w-action-post-url-value = '{{ view.get_enable_url }}'
+//  data-w-action-key-name-value = 'csrfmiddlewaretoken'
+//  data-action="click->w-action#enableAction"
+//  type="submit" class="button no">Enable</button>
+
+export class ActionController extends Controller {
+  static values = {
+    csrfToken: String,
+    postUrl: String,
```

Comment:
As per below comment - let's try doing just `url` as the value name. You could even set the default value to the current browser url. 

```suggestion
    url: { type: String, default: window.location.href },
```

Note. This will be the URL when the page loads - but should be sufficient for a default value for future use cases.